handwriting_recognition.ipynb

```diff
@@ -0,0 +1,1027 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.    train_paths, train_labels = prepare_image_paths(train_samples)
</pre><p>Similary, rename <code>train_paths</code>  to <code>train_img_paths</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/Handwriting-Recognizer-in-Keras/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>